libcst/codemod/visitors/_add_imports.py

```diff
@@ -19,17 +19,19 @@ class AddImportsVisitor(ContextAwareTransformer):
     """
     Ensures that given imports exist in a module. Given a
     :class:`~libcst.codemod.CodemodContext` and a sequence of tuples specifying
-    a module to import from as a string and optionally an object to import from
-    that module, ensures that that import exists. It will modify existing imports
-    as necessary if the module in question is already being imported from.
+    a module to import from as a string. optionally an object to import from
```

Comment:
Should "optionally" be capitalized?
```suggestion
    a module to import from as a string. Optionally an object to import from
```

Reply:
Oh yeah, good catch~